### ISSUE TITLE
Oauth failed. Oauth_problem=The signatire is invalid.

### ISSUE THREAD

**Issue by nigarjafar** [magento/magento2#19159]
### Preconditions (*)

1. Magento ver. 2.3.0-beta32
2. Set up and activated API Integration with full access

### Steps to reproduce (*)

1. Make POST call to /oauth/token/request

### Expected result (*)

1. Get request token 
_Doc says something like that:_
oauth_token=4cqw0r7vo0s5goyyqnjb72sqj3vxwr0h&oauth_token_secret=rig3x3j5a9z5j6d4ubjwyf9f1l21itrr

### Actual result (*)

1.  Signatire is invalid. I tried with and without checking **Add empty parameters to signature**
![image](https://user-images.githubusercontent.com/20247065/48317733-db1fea80-e60f-11e8-8591-8d4469f0d022.png)

2. 
![image](https://user-images.githubusercontent.com/20247065/48317807-1a9b0680-e611-11e8-9f7b-1a7a2a16fd4d.png)

**Comment by magento-engcom-team** [comment]
Hi @nigarjafar. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@nigarjafar do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
Hi @nigarjafar thank you for you report, this issue has already fixed in 2.3-develop but they you have another issue ->https://github.com/magento/magento2/issues/13961 so you can't take oauth token :)

**Comment by AlterWeb** [comment]
Hi, @engcom-backlog-nazar in witch version is this issue fixed? I think I have the same problem in version 2.3.1

**Comment by amitebizon** [comment]
Issue still persist in version 2.3.2

**Comment by michail1982** [comment]
Same in version 2.3.3

**Comment by ionnini** [comment]
Issue still persist in version 2.3.5
